docs/CodeDevelopment/compile.md

```diff
@@ -18,14 +18,13 @@ It is often desirable to compile test code and at times there might not be a pro
     Assumption is that the repository to compile has been cloned to your filesystem and is in the state ready to compile. 
 
 1. Open cmd prompt at workspace root
-2. Run 'mu_bootstrap` to get the necessary build components
 3. Run MuBuild to:
     * Clone code dependencies
     * Download binary dependencies
     * Statically test code
     * Compile test code
 ```cmd
-MU_BUILD\UefiBuild\MuBuild\MuBuild.py -c <Mu Repo Build Config File> -p <1st Mu Pkg Build Config File> <2nd Mu Pkg Build Config File...>
+mu_build-c <Mu Repo Build Config File> -p <1st Mu Pkg Build Config File> <2nd Mu Pkg Build Config File...>
```

Comment:
shouldn't that have a space between mu_build and -c?  I guess it doesn't matter completely since this isn't really a string that can be copied and used directly.